bin/improver-nbhood

```diff
@@ -93,6 +94,22 @@ def main():
                              'weighting is assumed. weighted_mode is only '
                              'applicable for calculating "probability" '
                              'neighbourhood output.')
+    parser.add_argument('--sum_or_fraction', default="fraction",
+                        choices=["sum", "fraction"],
+                        help='The neighbourhood output can either be in the '
+                             'form of a sum of the neighbourhood, or a '
+                             'fraction calculated by dividing the sum of the '
+                             'neighbourhood by the neighbourhood area. '
+                             '"fraction" is the default option.')
+    parser.add_argument('--re_mask', action='store_true',
+                        help='If re_mask is True, the original '
+                             'un-neighbourhood processed mask is applied to '
```

Comment:
I'm still finding this argument a bit confusing. I think it should be If re_mask option is set (i.e. True), ... If not set this defaults to False, the original ....